### DISCUSSION TITLE
What does 140% in the Chihiros app really mean?

### DISCUSSION THREAD

**Discussion by olakp** [TheMicDiet/chihiros-led-control#47] [General]
We have a Chihiros WRGB2 Pro and have been using this integration for some time now. Everything works absolutely perfectly, and I am so happy for the job made with this integration.
But something bugs my mind. In Chihiros app, the brightness can be set all the way up to 140%. Does anyone know what that means in reality? Is it really only full brightness, or is it "more" than full brightness? Or put it another way; does 255 on all channels equal "140%" in Chihiros app, or does it equal 100%? Why the app does not only use 0-100% is quite strange, of you ask me.
If anyone has insight in this, I'd be grateful.

**Comment by senna1992** [comment]
It is a feature that allows you to drive the color of your choice over its drivers rated power.

I never managed to get around this in full depth so I settled with the integrations 0-100% rating - and ditched the native app.


This is my take on explaining it:

Basically you can see this like a boost function for the driver. It puts out like 50w in total, since there are 4 colors this might be split in 4x10w and so 10w are left. These 10w are not evenly splitable under 4 loads so I guess they had to look for some good pitch iE 2w 0-2-4-6-8-10 or so.

After all the leds 100% is not the apps 100% to avoid too short lifetimes.

Hope this answers your question!




________________________________
From: olapettersson ***@***.***>
Sent: Sunday, November 10, 2024 10:06:45 PM
To: TheMicDiet/chihiros-led-control ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [TheMicDiet/chihiros-led-control] What does 140% in the Chihiros app really mean? (Discussion #47)


We have a Chihiros WRGB2 Pro and have been using this integration for some time now. Everything works absolutely perfectly, and I am so happy for the job made with this integration.
But something bugs my mind. In Chihiros app, the brightness can be set all the way up to 140%. Does anyone know what that means in reality? Is it really only full brightness, or is it "more" than full brightness? Or put it another way; does 255 on all channels equal "140%" in Chihiros app, or does it equal 100%? Why the app does not only use 0-100% is quite strange, of you ask me.
If anyone has insight in this, I'd be grateful.

**Reply by olakp** [reply]
Thanks! That makes sense. I haven't bothered to play around with the app, but I reckon that all 4 channels cannot be on 140% (and who would like that?), since then 140% should really be "100%". I am also guessing that you cannot set 140%, 100%, 100%, 100%, since then you should also be able to set 113%, 113%, 113%, 113% and consequently 113% should be the "100%". So you probably "steal" the extra % from the other channels, making the total always <= 400%.

EDIT: I went ahead and tried the app and it seems the max total wattage is always about 110 W for our WRGB2Pro. And that is really what my question is about. I wanted to create a wattage sensor for the light, but couldn't figure out if light level 255 equals 100% or 140% in the app. If it equals 140% in the app, will the light automatically limit the total output as to not burn the electronics, or is that protection really in the app, which will not allow all channels to be driven to 140% simultaneously? Because it is possible to run all channels on 255 via the integration? I guess I will have to plug the light through a power meter to really figure this out. 😁

**Reply by nf112-de** [reply]
I'm facing the same issue. It's not clear to me how I can translate the Chihiros app values (up to 140%) to this integrations values of up to 100%. This post seems to indicate that burning the Chips seems unlikely, but how do we verify that?
I'm using the Universal WRGB. 

@TheMicDiet Do you have an idea?
